src/components/molecules/index.js

```diff
@@ -1,14 +1,14 @@
-//> Components
```

Comment:
There are no molecules yet. Please remove the folder `src/components/molecules` and re-add it as soon as you create molecules.

Reply:
We preferred to create the structure with this branch, to make further contributing easier.